fastscanner/services/indicators/tests/test_subscribe_realtime.py

```diff
@@ -171,8 +187,11 @@ async def test_daily_atr_indicator(setup):
     )
     await indicator.extend(symbol, df)
 
-    channel_handler = CandleChannelHandler(symbol, [indicator], handler)
+    channel_handler = CandleChannelHandler(
```

Comment:
Because of the comment above, you don't need to add sleeps in the tests when freq is 1 min.